prototype/local-factory/main.go

```
@@ -0,0 +1,233 @@
+	}
+	cfg, err := loadConfig(*configPath)
+	if err != nil {
+		return err
+	}
+	body, _ := json.Marshal(map[string]string{"issue": reference})
+	client := http.Client{Timeout: 30 * time.Second}
+	response, err := client.Post(serverURL(cfg.Config.Server.Listen)+"/api/run", "application/json", bytes.NewReader(body))
+	if err != nil {
+		return fmt.Errorf("connect to factory web: %w", err)
+	}
+	defer response.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `response.Body.Close` is not checked

(errcheck)